mava/utils/jumanji_registry.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
```

Comment:
This file is highly tailored to be scenario-specific. which makes the integration of additional Jumanji environments complex as this will require each time extra environmental configuration functions and defining a range of attributes.

Reply:
Maybe instead we just add `constant.py `or `scenario.py` that includes the scenarios in constant dict 
```
{"2s-10x10-3p-3f":
  {
    "grid_size": 10
    "fov": 2
    "num_agents": 3
    "num_food": 3
    "max_agent_level": 2
    "force_coop": False
  }
}
```
Any additional thoughts from @RuanJohn and @sash-a could be helpful